Tools/WebKitTestRunner/ios/UIScriptControllerIOS.mm

```diff
@@ -860,8 +856,19 @@ static void clipSelectionViewRectToContentView(CGRect& rect, UIView *contentView
 JSObjectRef UIScriptControllerIOS::selectionEndGrabberViewRect() const
 {
     UIView *contentView = platformContentView();
-    UIView *selectionRangeView = [contentView valueForKeyPath:@"interactionAssistant.selectionView.rangeView"];
-    auto frameInContentViewCoordinates = [selectionRangeView convertRect:[[selectionRangeView valueForKeyPath:@"endGrabber"] frame] toView:contentView];
+    UIView *handleView = [contentView valueForKeyPath:@"interactionAssistant.selectionView.rangeView.endGrabber"];
+
+#if HAVE(UI_TEXT_SELECTION_DISPLAY_INTERACTION)
+    if (!handleView) {
+        // FIXME: We should be able to use -handleViews here, but this seems returns an empty array,
```

Comment:
This one is probably worth a UIKit bug.

Reply:
Good point! Filed rdar://108607881, and I'll link it here.